src/effects/effectslot.cpp

```diff
@@ -179,8 +181,6 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
             pParameter->loadEffect(pEffect);
         }
 
-        slotEffectMetaParameter(m_pControlMetaParameter->get(), true);
```

Comment:
If we remove the "true" this line does nothing and can be removed. 

Touching the meta-knob for the jump is required to signal that the meta-knob should be adopted and not the effect defaults.